doc/manpages/dinitctl.8.m4

```diff
@@ -298,6 +301,18 @@ The value can be provided on the command line or retrieved from the environment
 called in.
 Any subsequently started or restarted service will have these environment variables available.
 This is particularly useful for user services that need access to session information.
+.TP
+\fBsignal\fR
+Send a signal to a specified service.
+The \fIsignal-number\fR can either be specified as integer or signal name.
```

Comment:
"specifed as _an_ integer or signal name". Don't worry about this though, I will fix the grammar once it's merged (and probably change the wording a little).